src/Utils/samples.jl

```diff
@@ -120,63 +120,42 @@ end
 
 ## Rejection Sampling
 
-"""
-    _canonical_length(X::LazySet)
-
-Returns the support function of the given set along the positive and negative canonical directions.
-
-### Inputs
-
-- `X` - convex set
-
-### Outputs
-
-A matrix with `n = dims(X)` rows and two columns. Each row stands for
-one dimension of `X` whereas the first column is the minimum and the second
-column is the maximum value of the corresponding dimension.
-"""
-function _canonical_length(X::LazySet{N}) where {N<:Real}
-    dims = dim(X)
-    x = Matrix{N}(undef, dims, 2)
-    for j = 1:dims
-        ej = SingleEntryVector(j, dims, one(N))
-        x[j,:] = [-ρ(-ej, X), ρ(ej, X)]
-    end
-    return x
-end
-
 """
     sample(X::LazySet{N}, num_samples::Int;
            rng::AbstractRNG=GLOBAL_RNG,
            seed::Union{Int, Nothing}=nothing) where {N}
 
-Rejection sampling of an arbitrary LazySet `X` for which the support value
-function is defined.
-
-Draw a sample `x` from a uniform distribution of a box-overapproximation of the
-original set `X` in all `n` dimensions. The function rejects a drawn sample `x`
-and redraws as long as the sample is not contained in the original set `X`,
-i.e., `x ∉ X`.
+Sampling of an arbitrary `LazySet` `X`.
 
 ### Input
 
 - `X`           -- lazyset
-- `num_samples`    -- number of random samples
+- `num_samples` -- number of random samples
+- `sampler`     -- Sampler used (default:`RejectionSampler`)
 - `rng`         -- (optional, default: `GLOBAL_RNG`) random number generator
 - `seed`        -- (optional, default: `nothing`) seed for reseeding
 
 ### Output
 
 A vector of `num_samples` vectors.
 If `num_samples` is not passed one sample as a single vector is returned.
+
+### Algorithm
+`RejectionSampler`:
+Draw a sample `x` from a uniform distribution of a box-overapproximation of the
+original set `X` in all `n` dimensions. The function rejects a drawn sample `x`
+and redraws as long as the sample is not contained in the original set `X`,
+i.e., `x ∉ X`.
```

Comment:
I suggest to use double ticks for math mode here (as in \`\`x\`\`) and to move this part to the documentation of `RejectionSampler` instead.